platform-sdk/swirlds-common/src/main/java/com/swirlds/common/context/DefaultPlatformContext.java

```diff
@@ -33,25 +35,51 @@ public final class DefaultPlatformContext implements PlatformContext {
     private final Metrics metrics;
     private final Cryptography cryptography;
     private final Time time;
+    private final FileSystemManager fileSystemManager;
 
     /**
      * Constructor.
      *
-     * @param configuration the configuration
-     * @param metrics       the metrics
-     * @param cryptography  the cryptography
-     * @param time          the time
+     * @param configuration     the configuration
+     * @param metrics           the metrics
+     * @param cryptography      the cryptography
+     * @param time              the time
      */
```

Comment:
We should probably stop trying to align these, it creates a lot of comment churn.